README.md

```diff
@@ -480,6 +480,57 @@ different version number, and also marked `default-arg-value`:
 This is due to the translation of defaults into the CFG: there is a synthetic conditional that chooses either to
 initialize the variable from the argument passed at the send, or to the default value when no value is present.
 
+Finding all references works differently in package specification (__package.rb) files. Consider the following:
+
+```ruby
+class Foo < PackageSpec
+  import Bar
+```
+
+Calling "find all references" on `Bar` in this file will return only references to `Bar` in the `Foo` package. LSP tests 
```

Comment:
What does this mean Go to Definition will report for `import Bar` here?

Reply:
Go to definition on `import Bar` will go to the declaration of the `Bar` package -- that behavior doesn't change.